packages/ui/src/app/components/FMBanner.tsx

```diff
@@ -62,9 +72,13 @@ const StyledLink = styled(Link)`
   font-size: 14px;
 `
 
+const ButtonLink = styled(Link)`
+  text-decoration: none;
+  color: inherit;
+`
+
 const Container = styled.div<{ open: boolean }>`
-  /* display: ${({ open }) => !open && 'none'}; */
-  display: none;
+  display: ${({ open }) => !open && 'none'};
   background-color: ${Colors.Black[800]};
   border-radius: ${BorderRad.l};
   width: 80%;
```

Comment:
Also lets center the modal:
```suggestion
  left: 246px;
```